plugins/config/src/FromConfigAdapter/FromConfigSequenceAdapter.ts

```diff
@@ -32,9 +33,10 @@ export default class FromSequenceConfigAdapter extends FromConfigAdapter {
         observer.next(
           new SimpleFeature({
             ...feat.toJSON(),
+            end: region.end,
+            start: region.start,
             seq,
-            end: featStart + seq.length,
-            start: featStart,
+            uniqueId: `${Math.random()}`,
```

Comment:
maybe use something other than random, cause it's not reproducible. maybe do a simple fast hash of something. crc32 of a concatenation of some of the data maybe

Reply:
actually maybe it should just be `feat.id() + ':' + region.start + '..' + region.end`